### ISSUE TITLE
Paypal Integration

### ISSUE THREAD

**Issue by anu0012** [tylerjrichards/st-paywall#77]
Hi there! Will there be any Paypal integration in the near future?

**Comment by tylerjrichards** [comment]
no, I don't think so. I'd encourage people to use Stripe over Paypal anyway! Any reason not to use Stripe?
